extra/1-radio-stations.js

```diff
@@ -13,6 +15,14 @@
  * - Should return this array to use in other functions
  */
 
+function getAllFrequencies() {
+	let frequencies = [];
+	for (let i = 87; i < 109; i++) {
```

Comment:
Maybe assign 87 and 109 to variables called something like startFrequency and endFrequency to make the code more understandable